sysl2/sysl/examples/grammars/go/go.gen.sysl

```diff
@@ -26,16 +32,19 @@ CodeGenTransform:
   !view GoStructFields(fields <: set of Type) -> set of FieldDecl:
     fields -> (field:
       let isKeyword = IsKeyword(field.key).out
-      let prefix = if isKeyword == true then "_" else ""
-      identifier = prefix + field.key
+      let fieldName = GetGoField(field.key).out
+      identifier = fieldName
       let IsPtr = if field.value.optional == true then "*" else ""
       Type = IsPtr + GoType(field.value).out
+      let fieldTag = '`json:"' + ToLower(fieldName)
+      Tag = if field.value.optional == true then (fieldTag + ',omitempty"`') else (fieldTag + '"`')
     )
 
   !view GoStructs(types <: set of Type) -> set of TopLevelDecl:
     types -> (type:
         Declaration = type -> <Declaration>(:
           StructType = type -> <StructType>(:
+            Comment = "\n//" + .key + " ..." 
```

Comment:
in current form it would be better not to have comment, if comment is just "<StructName> ...".

Reply:
It's to shut the linter up.